meta-phosphor/recipes-phosphor/inventory/phosphor-inventory-manager_git.bb

```diff
@@ -27,6 +27,8 @@ DEPENDS += " \
         ${PYTHON_PN}-mako-native \
         "
 
+RDEPENDS_${PN} += "bash"
```

Comment:
Any reason it couldn't just use sh? It's even called clear-all-faults.sh

Reply:
no particular reason. The same script may be needed to add more support in future & /bin/sh may not be sufficient enough so, used bash.

Reply:
I have removed it and kept it as sh instead of bash